mypy/build.py

```diff
@@ -37,12 +37,13 @@
 from mypy import util
 from mypy.fixup import fixup_module_pass_one, fixup_module_pass_two
 from mypy.nodes import Expression
-from mypy.options import Options
 from mypy.parse import parse
 from mypy.stats import dump_type_stats
 from mypy.types import Type
 from mypy.version import __version__
 
+if TYPE_CHECKING:
```

Comment:
Oops. Yes, I definitely missed the earlier comment. I'll fix the uses of `TYPE_CHECKING` that I aded.